cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/task/RunExecutableTask.java

```diff
@@ -151,13 +156,16 @@ private void runGeneratedExecutable(BLangPackage executableModule, BuildContext
 
         ExecutableJar executableJar = buildContext.moduleDependencyPathMap.get(executableModule.packageID);
         String initClassName = BFileUtil.getQualifiedClassName(executableModule.packageID.orgName.value,
-                                                               executableModule.packageID.name.value,
-                                                               MODULE_INIT_CLASS_NAME);
+                executableModule.packageID.name.value, MODULE_INIT_CLASS_NAME);
         try {
             List<String> commands = new ArrayList<>();
             commands.add("java");
             commands.add("-cp");
             commands.add(getClassPath(executableJar));
+            if (isInDebugMode) {
```

Comment:
RunExecutableTask can have debugport pass through runcommand and then we can check that here.

Reply:
https://github.com/ballerina-platform/ballerina-lang/pull/20087#discussion_r351645513